packages/stems/src/components/SelectablePill/HarmonySelectablePill.tsx

```diff
@@ -0,0 +1,37 @@
+import { forwardRef } from 'react'
+
+import cn from 'classnames'
+
+import styles from './HarmonySelectablePill.module.css'
+import { HarmonySelectablePillProps } from './types'
+
+export const HarmonySelectablePill = forwardRef<
+  HTMLButtonElement,
+  HarmonySelectablePillProps
+>((props, ref) => {
+  const {
+    size,
+    isSelected,
+    label,
+    icon: IconComponent,
+    className,
+    ...restProps
+  } = props
+  return (
+    <button
+      className={cn(
+        styles.pill,
+        {
+          [styles.large]: size === 'large',
+          [styles.selected]: isSelected
+        },
+        className
+      )}
+      ref={ref}
+      {...restProps}
+    >
+      {IconComponent == null ? null : <IconComponent className={styles.icon} />}
```

Comment:
===

Reply:
I read this as intending to catch `null` or `undefined`. But I think the general pattern has been to just booleanize things? As there aren't really any values you can pass as a node which are falsey and we would actually _want_ to render.
`{IconComponent ? <IconComponent ... /> : null}`

Reply:
ohh i see! yeah i prefer your last option (though @nicoback likes to have null come first, which i can get behind) what do you think @nicoback ?